### PR TITLE
Adding notes about jumphosts.

### PR DESCRIPTION
**Pull request by benarent**
Content for ProxyJump #2882 

Ref: https://github.com/gravitational/teleport/issues/2543 

I've added in updated output from the `tsh` and `teleport` CLI

### CODE REVIEW

File: docs/4.0/user-manual.md [hunk 1]
```diff
@@ -343,6 +356,19 @@ This command:
...
 
+
+### SSH Jumphost
+While implementing ProxyJump for Teleport, we have extended the feature to `tsh`. 
+
+`$ tsh ssh -J proxy.example.com telenode`
+
+Known limits:
+- Only one jump host is supported (`-J` supports chaining that teleport does not utilise) 
+and tsh will return with error in case of two jumphosts: `-J` a,b will not work.
```
Comment (webvictim) [inline docs/4.0/user-manual.md]: `-J a,b` (extend the code example to include the two arguments)

File: docs/4.0/user-manual.md [hunk 2]
```diff
@@ -343,6 +356,19 @@ This command:
...
 3. Executes `curl` command locally, which results in `curl` hitting google.com:80 via `node`
 
+
+### SSH Jumphost
+While implementing ProxyJump for Teleport, we have extended the feature to `tsh`. 
+
+`$ tsh ssh -J proxy.example.com telenode`
+
+Known limits:
+- Only one jump host is supported (`-J` supports chaining that teleport does not utilise)
```
Comment (webvictim) [inline docs/4.0/user-manual.md]: `teleport` -> `Teleport`

File: docs/4.0/user-manual.md [hunk 3]
```diff
@@ -343,6 +356,19 @@ This command:
...
+While implementing ProxyJump for Teleport, we have extended the feature to `tsh`. 
+
+`$ tsh ssh -J proxy.example.com telenode`
+
+Known limits:
+- Only one jump host is supported (`-J` supports chaining that teleport does not utilise) 
+and tsh will return with error in case of two jumphosts: `-J` a,b will not work.
+
+ -When `tsh ssh -J user@proxy` is used, it overrides the SSH proxy defined in the tsh 
+profile and port-forwarding is used instead of the existing teleport proxy subsystem.
```
Comment (webvictim) [inline docs/4.0/user-manual.md]: `port-forwarding` -> `port forwarding`

`teleport` -> `Teleport`

**Review by webvictim — APPROVED** [review]
LGTM with a few notes

**Comment by benarent** [comment]
Thanks Gus!

**Pull request merged**